Packet++/header/CiscoHdlcLayer.h

```diff
@@ -0,0 +1,103 @@
+#pragma once
+
+#include "Layer.h"
+
+namespace pcpp
+{
+	/// @class CiscoHdlcLayer
+	/// Represents a Cisco HDLC protocol layer
+	class CiscoHdlcLayer : public Layer
+	{
+	public:
+		/// @enum Address
+		/// Represents Cisco HDLC address types
+		enum class Address
```

Comment:
`Address` is misleading. I would suggest `AddressType`.

Reply:
That's how it shows in Wireshark:
![image](https://github.com/user-attachments/assets/c92a16e4-2fcf-4d3b-8053-5c8319e09bfc)

And in Wikipedia: 
![image](https://github.com/user-attachments/assets/ca56fd94-cc8e-4f1d-9e32-24180b80c4fc)

Reply:
Updated in this commit: https://github.com/seladb/PcapPlusPlus/pull/1747/commits/c933346a1386c1657c2548a82916b7263e7a3659